doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -132,6 +132,13 @@ Matter
   * Page about :ref:`ug_matter_device_optimizing_memory`.
   * Shell commands for printing and resetting the peak usage of critical system resources used by Matter.
     These shell commands are available when both :kconfig:option:`CHIP_LIB_SHELL` and :kconfig:option:`CHIP_STATISTICS` Kconfig options are set.
+  * Reaction to the last fabric removal.
+    Now the user can decide what should happen after removing the last fabric:
```

Comment:
```suggestion
    The user now decides what happens after the removal:
```
"can decide" instead of "now decides" if there is a default, in which case the default should be noted for that entry.